tests/test.cfg

```diff
@@ -141,13 +142,13 @@ from-date = 2017-01-01
 [jenkins]
 raw_index = jenkins_test-raw
 enriched_index = jenkins_test
-fetch-cache = true
+no-cache = true
```

Comment:
`no-cache` and `fetch-cache` are different options which are not semantically equivalent.